sections/introduction.tex

```diff
@@ -4,57 +4,46 @@ \section{Introduction}\label{sec:introduction}
 When referring to a dataset, we talk about a dataset including metadata information, which are non-essential data features.
 One example of this could be the publication date of a text document. 
 In most contexts, it is not needed to use and understand the document; however, it provides more context to the main content of the document: the text.
+
+% Intro to Nordjyske data and applications
 However, there is usually a lot of data, that is not used, either because it is not useful or we do not know how to use it properly.
```

Comment:
```suggestion
In most contexts, it is not needed to use and understand the document; however, it provides more context to the main content of the document: the text.
However, a lot of metadata is not used, either because it is not useful or we do not know how to use it properly.

% Intro to Nordjyske data and applications
```